benchmarks/IO.py

```diff
@@ -232,9 +286,11 @@ def create_parser():
     comp_str = args.compression
     comp_types = COMPRESSIONS if comp_str == "" else comp_str.lower().split(",")
 
+    fileFormat = FileFormat.CSV if args.csv else FileFormat.PARQUET if args.parquet else FileFormat.HDF5
```

Comment:
I think this is an issue if both Parquet and CSV are set.

I think maybe what I would do is run both CSV and Parquet if they are both set? But that also seems odd, so I'm not sure... 

Anyways, I think this is fine for now, as it is mainly run by our nightly performance testing, but I think that could potentially cause confusion

Reply:
Hi Ben, the argument parser will not allow both of them to be set. In lines 235-241, the two arguments are in a mutually exclusive group now; therefore, only one of them is allowed at a time.

Reply:
Great, thanks for clarifying!